project_redss/redss_code_translators.py

```diff
@@ -0,0 +1,24 @@
+from core_data_modules.cleaners import CodeTranslator, Codes
```

Comment:
Is it possible to parse these rather than coding them?

Reply:
Yes, but I thought we weren't to rely on/use the display text for anything

Reply:
Ah yeah, so if we are going to code them like I'd like to ensure that these codes all exist in the scheme.

The risk I'm concerned about is that someone makes a mistake, regenerates the scheme with new IDs and starts using it. Is there anything in the pipeline to detect that we're attempting to interpret data with the wrong coding scheme?

Reply:
Still not sure what to do here. Even with a check, maintaining those tables is going to be a pain

Reply:
Will change to put this data in the scheme in a future PR